plop-templates/@react-stately/docs/useComponentState.mdx.hbs

```diff
@@ -37,7 +37,7 @@ keywords: []
 
 ## Options
 
-<ClassAPI links={docs.links} class={docs.links[docs.exports.use{{~componentName~}}State.parameters[0].value.base.id]} />
+// <ClassAPI links={docs.links} class={docs.links[docs.exports.use{{~componentName~}}State.parameters[0].value.base.id]} />
```

Comment:
why is this commented out?

Reply:
looks like you can ignore this comment

Reply:
Sorry, just realized the comment doesn't actually work so just deleted it since it breaks docs build on fresh plop.